includes/User.php

```diff
@@ -131,11 +131,12 @@ class User {
 
 	/** @name Cache variables */
 	//@{
-	var $mId, $mName, $mRealName, $mNewpassword, $mNewpassTime,
+	private $mRealName, $mNewpassword, $mNewpassTime,
 		$mEmail, $mTouched, $mToken, $mEmailAuthenticated,
-		$mEmailToken, $mEmailTokenExpires, $mRegistration, $mGroups, $mOptionOverrides,
-		$mCookiePassword, $mEditCount, $mAllowUsertalk;
-	var $mBirthDate; // Wikia. Added to reflect our user table layout.
+		$mEmailToken, $mEmailTokenExpires, $mRegistration, $mGroups,
+		$mOptionOverrides, $mEditCount, $mAllowUsertalk;
+	private $mBirthDate; // Wikia. Added to reflect our user table layout.
+	var $mId, $mName;
 	//@}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)